textproc/sentencepiece/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        google sentencepiece 0.1.84 v
+revision            0
+categories          textproc
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Unsupervised text tokenizer for Neural Network-based\
+                    text generation.
+
+long_description    SentencePiece is an unsupervised text tokenizer\
+                    and detokenizer mainly for Neural Network-based\
+                    text generation systems where the vocabulary size\
+                    is predetermined prior to the neural model\
+                    training. SentencePiece implements subword units\
+                    (e.g., byte-pair-encoding (BPE) \[Sennrich et al.\])\
+                    and unigram language model \[Kudo.\]) with the\
+                    extension of direct training from raw\
+                    sentences. SentencePiece allows us to make a\
+                    purely end-to-end system that does not depend on\
+                    language-specific pre/postprocessing.
+
+homepage            https://github.com/google/sentencepiece
```

Comment:
this is the default set by the github PG, so this line should be removed

Reply:
Thanks done.